src/legacy/ui/public/vis/editors/default/agg.html

```diff
@@ -86,21 +96,21 @@
         tooltip="{{::'common.ui.vis.editors.agg.removeDimensionButtonTooltip' | i18n: { defaultMessage: 'Remove Dimension' } }}"
         tooltip-append-to-body="true"
         type="button"
-        class="kuiButton kuiButton--basic kuiButton--small">
+        data-test-subj="removeDimension-{{describe()}}"
```

Comment:
I'm not completely sure if this is a good ID for that. We should keep the data-test-subj attribute as much as possible static. In this case if describe change it's implementation will fail the test (and we are not testing the describe method here).
I think you can work the same way is done with the `disableAggregationBtn` and it's code in the visualize page object
```
async toggleDisabledAgg(agg) {
      await testSubjects.click(`aggregationEditor${agg} disableAggregationBtn`);
      await PageObjects.header.waitUntilLoadingHasFinished();
    }
```